frontend/src/components/CardDetailsPage.tsx

```
@@ -38,11 +41,46 @@ const DetailsCard = ({
+
+          <div className="col-span-5">
+            {type === 'user' && (
+              <SecondaryCard
+                  title={`${type[0].toUpperCase() + type.slice(1)} Details`}
+                  className={'md:col-span-5 gap-2'}
+              >
+                {details && details.map((detail, index) => (
+                  <div key={index}>
+                    <strong>{detail.label}:</strong> {detail.value ? detail.value : 'Unknown'}
+                  </div>
+                  ))}
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)